Add basic Kubernetes Helm chart

Calebjh opened the pull request:
Are TLS cert files expected to be included in the image itself, or should I add support to include those as part of the deployment?

Resolves #30

palmerabollo commented:
Very good work, Caleb.

SSL support was included by @jarett-cyxtera on #27.

Without using helm, you usually have a kubernetes deployment with a volume containing the cert files mounted. They are externally provided to prometheus-kafka-adapter because they are different in every case (they can not be included in the image itself).

I think we should support that, yes.

Calebjh commented:
Okay, I fixed the TLS/SSL values as described in #32, because I pulled the values from the README initially, and added the ability to set the SSL secrets as part of the deployment (with some default file paths), or the paths can still be specified without secrets in case anybody wants to build custom, private images with their certs baked in.
